.github/workflows/ci.yml

```diff
@@ -22,8 +33,6 @@ jobs:
         ports:
           - 5672:5672
       minio:
-        # This image does not require any command arguments (which GitHub Actions don't support)
-        # TODO at the time this was pinned, latest bitnami/minio was broken
```

Comment:
Oh you're right. In a previous commit, I bumped it to latest and removed the comment, but then in my last commit, I pinned it again because the latest minio still breaks our `check-migrations` test. It at least was a good checkpoint for whether we still need it pinned (we do). So I'll put the comment back in, too.

Reply:
Changed in [fd836a4](https://github.com/OpenImaging/miqa/pull/468/commits/fd836a4d9c7a9288f537a1898015fafc1eec289f). I didn't add back the first line of that comment, though, because it seems unnecessary. The other services don't have command arguments either.